qiskit/providers/__init__.py

```diff
@@ -612,6 +612,35 @@ def result(self):
         def status(self):
             return JobStatus.DONE
 
+Primitives
+==========
+
+While not directly part of the provider interface the :mod:`qiskit.primitives`
+module is tightly coupled with providers. Specifically the primitive
+interfaces, such as :class:`~.BaseSampler` and :class:`~.BaseEstimator`,
+are designed to enable provider implementations to provide custom
+implementations which are optimized for the provider's backends. This can
+include customizations like circuit transformations, additional pre and
+post-processing, batching, caching, error mitigation, etc. The concept of
+the :mod:`qiskit.primitives` module is to explicitly enable this as the
+primitive objects are higher level abstractions to produce processed higher
+level outputs (such as probability distributions and expectation values)
+that abstract away the mechanics of getting the best result efficienctly to
+concentrate on higher level applications using these outputs.
+
+For example, if you're backends were well suited to leverage
+`mthree <https://github.com/Qiskit-Partners/mthree/>`__ measurement
+mitigation to improve the quality of the results you could implement a
+provider specific :class:`~.Sampler` implementation that leverages the
```

Comment:
`Sampler` or `BaseSampler`?

Reply:
Yeah it should be `BaseSampler` 

```suggestion
provider specific :class:`~.BaseSampler` implementation that leverages the
```